### PR TITLE
Feat/cheatsheet

### PR DESCRIPTION
**Pull request by Her0elt**
Cheatsheet site 
Lint, test and build
## Description
Link to Trello task: https://trello.com/c/iQwmlmtA

Comments/issues/screenshots:
The search function does not work, something in backend is not working. Breadcrums are commented out because react can't find a module with that name. There is also only a function for GET request for backend to cheatsheet, that gets all cheatsheets. Because of these missing elements cheatsheet is not added to the sidebar or navigation bar.

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] PR name is descriptive
- [ ] The PR includes the link to the relevant Trello task
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

- [ ] `yarn build` was run locally without errors
- [ ] `yarn test` has passed locally and any fixes were made for failures
- [ ] `yarn format` was run locally and fixes were made for new errors

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: src/containers/Cheatsheet/index.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+          <div className={classes.folders}>
+            <LinkButton className={classes.folders}to ={URLS.cheatsheet+''.concat(getStudy(4), '/')}>
+              <Typography className={classes.text}>{getUserStudyLong(4)}</Typography>
+            </LinkButton>
+          </div>
+          <div className={classes.folders}>
+            <LinkButton className={classes.folders}to ={URLS.cheatsheet+''.concat(getStudy(5), '/')}>
+              <Typography className={classes.text}>{getUserStudyLong(5)}</Typography>
+            </LinkButton>
+          </div>
```
Comment (olros) [inline src/containers/Cheatsheet/index.js:121]: Du kan bruke .map() for å slippe å skrive disse 5 ganger

File: src/containers/Cheatsheet/index.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+          </Link>
+          <Link
+            color="textPrimary"
+            href="/components/breadcrumbs/"
+            onClick={handleClick}
+            aria-current="page"
+          >
+                    Breadcrumb
+          </Link>
+        </Breadcrumbs>*/}
```
Comment (olros) [inline src/containers/Cheatsheet/index.js:95]: Har du prøvd Breadcrumbs etter at vi oppdaterte Material-UI?

File: src/index.js [hunk 3]
```diff
@@ -123,6 +127,10 @@ const Application = (
           <Route path={URLS.privacyPolicy} component={PrivacyPolicy} />
           <Route path={URLS.eventRules} component={EventRules} />
 
+          <Route exact path={URLS.cheatsheet} component={Cheatsheet}/>
+          <Route path={URLS.cheatsheet.concat(':studyId/:classId/')} component={filesCheatsheet}/>
+          <Route path={URLS.cheatsheet.concat(':id/')} component={classesCheatsheet}/>
```
Comment (olros) [inline src/index.js:132]: Noe grunn til at det er `:studyId` på files og bare `:id` på classes?

File: src/index.js [hunk 4]
```diff
@@ -40,6 +40,10 @@ import EventRegistration from './containers/EventRegistration';
 import UserService from './api/services/UserService';
 import SignUp from './containers/SignUp';
 import PrivacyPolicy from './containers/PrivacyPolicy';
+import Cheatsheet from './containers/Cheatsheet';
+import classesCheatsheet from './containers/Cheatsheet/components/Classes';
+import filesCheatsheet from './containers/Cheatsheet/components/Files';
```
Comment (olros) [inline src/index.js:45]: Gjerne bruk store forbokstav på components

File: src/containers/Cheatsheet/index.js [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+const getStudy = (i) => {
+  switch (i) {
+    case 1: return 'DATAING';
+    case 2: return 'DIGFOR';
+    case 3: return 'DIGSEC';
+    case 4: return 'DIGSAM';
+    case 5: return 'DRIFT';
+    default: return '?';
+  }
+};
```
Comment (olros) [inline src/containers/Cheatsheet/index.js:72]: Kan du bruke `getUserStudyShort` fra utils her?

File: src/containers/Cheatsheet/components/Classes.js [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+const getClass = (i) => {
+  switch (i) {
+    case 1: return 'FIRST';
+    case 2: return 'SECOND';
+    case 3: return 'THIRD';
+    case 4: return 'FOURTH';
+    case 5: return 'FIFTH';
+    default: return '?';
+  }
+};
```
Comment (olros) [inline src/containers/Cheatsheet/components/Classes.js:65]: Disse bør kanskje være på norsk?

Comment (Her0elt) [reply src/containers/Cheatsheet/components/Classes.js:65]: trenger de pågrunn av backend url

File: src/containers/Cheatsheet/index.js (hunk 5 above)
Comment (Her0elt) [reply src/containers/Cheatsheet/index.js:72]: samme her trenger de sånn pågrunn av backend, i backend så har enumen store bokstaver

File: src/index.js [hunk 7]
```diff
@@ -128,8 +128,8 @@ const Application = (
           <Route path={URLS.eventRules} component={EventRules} />
 
           <Route exact path={URLS.cheatsheet} component={Cheatsheet}/>
-          <Route path={URLS.cheatsheet.concat(':studyId/:classId/')} component={filesCheatsheet}/>
-          <Route path={URLS.cheatsheet.concat(':id/')} component={classesCheatsheet}/>
+          <Route path={URLS.cheatsheet.concat(':studyId/:classId/')} component={FilesCheatsheet}/>
+          <Route path={URLS.cheatsheet.concat(':id/')} component={ClassesCheatsheet}/>
```
Comment (olros) [inline src/index.js:132]: `:id` må vel byttes ut med `:studyId` nå?

**Pull request merged**